docs/dev-env-first-time-contributors.md

```diff
@@ -294,7 +294,24 @@ Welcome to Ubuntu 14.04.1 LTS (GNU/Linux 4.4.0-21-generic x86_64)
 Congrats, you're now inside the Zulip dev environment!
 
 You can confirm this by looking at the command prompt, which starts with
-`(zulip-venv)`.
+`(zulip-venv)`. In case it does not, your development environment might be haulted
+while provisioning the vagrant guest. Check the `/var/log/zulip_provision.log`
+file to find it ending with something similar to the following lines
+in case of successful completion:
+
+```
+....
+^[[94mZulip development environment setup succeeded!^[[0m
```

Comment:
Here I would only include the line "Zulip development environment setup succeeded!" and without the extra control characters.